backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java

```diff
@@ -16,9 +16,10 @@
 import com.virtuslab.gitcore.api.IGitCoreCommit;
 import com.virtuslab.gitcore.api.IGitCoreLocalBranchSnapshot;
 import com.virtuslab.gitcore.api.IGitCoreRemoteBranchSnapshot;
+import com.virtuslab.gitmachete.backend.api.RelationToRemote;
 import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
 
-public class GitMacheteRepository_deriveSyncToRemoteStatusUnitTestSuite extends BaseGitMacheteRepositoryUnitTestSuite {
+public class GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite extends BaseGitMacheteRepositoryUnitTestSuite {
```

Comment:
Whoops too much stuff in this name

Reply:
Should it be called `GitMacheteRepository_deriveSyncToRemoteStatusUnitTestSuite` or `GitMacheteRepository_deriveRelationToRemoteUnitTestSuite` now?

Reply:
`GitMacheteRepository_deriveRelationToRemoteUnitTestSuite` (current name is okay)